decentraland/unity-explorer
# Fix: nft instantiating and appearing

## What does this PR change?

Fixes the PBNftShape and NftShapeComponent:

1. PBNftShape is added to ComponentsContainer
2. URN parsing, added '{chain}' to the parsing template
3. Proper Initialize in NFTShapePlugin with settings included
4. 8 MB max size for nft image
5. InstantiateNftShapeSystem fixes
6. Updates for UrnToUrlTest
7. New tool in debug panel "Show all entities"

## How to test the changes?

1. Go to scene 82, -1
2. See a frog picture (in theory there should be more nfts, but they won't be loaded for other issues that we have to investigate)
![Uploading image.png…]()


extra info in the ticket: https://github.com/decentraland/unity-explorer/issues/1093

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
